### ISSUE TITLE
Create a Contributing.md

### ISSUE THREAD

**Issue by PaliC** [pytorch/multipy#252]
TL;DR: Build `torch::deploy` in OSS and run unit tests + document what you did.

Our Contributing.md was forgotten and is at the point where it ["exists"](https://github.com/pytorch/multipy/blob/main/CONTRIBUTING.md). We have gotten some complaints from developers that our development workflow instructions are not great. Fortunately, this presents a pretty good learning opportunity for newer developers. 
- [ ] Get on a Debian based system as that will make things easier.
- [ ] Clone the repo using a standard git workflow
- [ ] Follow the instructions for the [Pip Installation](https://github.com/pytorch/multipy/blob/main/CONTRIBUTING.md) up until actually running pip install then run these [instructions to build](https://github.com/pytorch/multipy/blob/main/CONTRIBUTING.md)
- [ ] Document this build process in `CONTRIBUTING.md`
- [ ] Run our [unit tests](https://github.com/pytorch/multipy#running-unit-tests-for-multipyruntime)
- [ ] Document this build process in our CONTRIBUTING.md
- [ ] Make some temporary changes into `multipy/runtime/deploy.cpp` (these can be print statements/ asserts/ whatever just don't commit them).
- [ ] Test to confirm that your changes have occurred, and document the process in `CONTRIBUTING.md`.
- [ ] Make some temporary changes into `multipy/runtime/interpreter/interpreter_impl.cpp` (these can be print statements/ asserts/ whatever just don't commit them).
- [ ]  Test to confirm that your changes have occurred, and document the process in `CONTRIBUTING.md` and mention that this process is only needed for changes that are specific to individual interpreters.
- [ ] [Create a Pull Request](https://opensource.com/article/19/7/create-pull-request-github) and press the button on the PR to import it into FBCode. The only file that is changed should be `CONTRIBUTING.md`, and submit it for review.

I expect there should be some trial and error with some of these steps. When you are recording things into the `CONTRIBUTING.md` think about how a user would avoid those situations. If you get stuck anywhere feel free to reach out to Sahan Paliskara.
